frontend/src/components/Overview.module.scss

```diff
@@ -34,10 +34,11 @@
   }
   div {
     textarea {
-      overflow: scroll !important;
+      overflow: auto !important;
+      background-color: $COLOR_WHITE;
     }
     div:nth-child(2) {
-      padding-right: 15%;
+      padding-right: 5%;
       justify-content: left;
     }
```

Comment:
What is this supposed to target?
Whatever it is, I think this is matching some unwanted elements as well.

I'd also prefer a fixed (`px`, `rem`, etc.) padding instead of relative (`%`), especially as the targeted element isn't clear.

Reply:
It targets the edit button so there's gap after the edit button but now I realized I could just give padding-right: 10px; for the .editableContainer and it does the job